### PR TITLE
localprovider doc update

### PR DESCRIPTION
**Pull request by a7vicky**

### CODE REVIEW

File: docs/configmanager.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Currently, the following sources are supported:
 | Source | Description |
 | --- | --- |
 | `OCM` | Retrieve an UpgradeConfig from the OpenShift Cluster Manager [`upgrade_policies`](https://api.openshift.com/#/default/get_api_clusters_mgmt_v1_clusters__cluster_id__upgrade_policies) API |
+| `LOCAL` | Using UpgradeConfig CR localy on the OpenShift Cluster|
```
Comment (dofinn) [inline docs/configmanager.md:13]:
```suggestion
| `LOCAL` | Using UpgradeConfig CR locally on the OpenShift Cluster|
```

File: docs/configmanager.md [hunk 2]
```diff
@@ -38,3 +39,13 @@ configManager:
...
 ```
+
+### LOCAL UpgradeConfig Manager
+
+The following configuration fields must be set:
+
+| Field | Description | Example |
+| --- | --- | --- |
+| `source` | Indicates the type of config manager being used | `LOCAL` |
+| `LocalConfigName` | Name of the Local config being used | `example-upgrade-config` |
```
Comment (dofinn) [inline docs/configmanager.md:50]: @mrbarge was the config a singleton? "managed-upgrade-config" ?

Comment (a7vicky) [reply docs/configmanager.md:50]: @dofinn Thanks for review. Yes it seems [upgradeconfig CR name](https://github.com/openshift/managed-upgrade-operator/blob/3518984ff470946edfb8156f1ac388c4c0be98ec/pkg/localprovider/localprovider.go#L14) (managed-upgrade-config) is fixed which is looked by MUO.

https://github.com/openshift/managed-upgrade-operator/blob/master/pkg/localprovider/localprovider.go#L63-L80

File: docs/configmanager.md (hunk 1 above)
Comment (a7vicky) [reply docs/configmanager.md]: @dofinn  I have fixed the typo.

**Pull request merged**